src/main/java/dao/Storage.java

```diff
@@ -0,0 +1,7 @@
+package dao;
+
+import java.util.HashMap;
+
+public class Storage {
+    public static final HashMap<String, Integer> storage = new HashMap<>();
```

Comment:
According to the encapsulation principle, the `storage` HashMap should not be public. Exposing it allows for modification from anywhere in the code, which can lead to hard-to-track bugs and security issues. Consider using a private modifier and providing controlled access through public methods.